api/controllers/console/app/statistic.py

```diff
@@ -65,13 +65,13 @@ def get(self, app_id):
         with db.engine.begin() as conn:
             rs = conn.execute(db.text(sql_query), arg_dict)
 
-        response_data = []
+            response_data = []
```

Comment:
Defining `response_data` inside the `with` statement might cause issues when trying to access it in the final `return` statement. Put it above `with` would be better.

Reply:
The changes below are the same, you know. 
Whatever, thank you very much for your contribution! 😊

Reply:
I have fixed it